src/biogeochem/CNGapMortalityMod.F90

```diff
@@ -489,5 +606,23 @@ subroutine CNGap_PatchToColumn (bounds, num_soilc, filter_soilc, &
     end associate
 
   end subroutine CNGap_PatchToColumn
+!    subroutine vegc_gmtransfer(ito,ifrom,default_transfer,matrix_transfer,vegcpool,gm_rate)
```

Comment:
Please clean up all your files for this pull request. Especially, it seems like lots of commented out codes are still in the files...